File: Sportlance.WebUI/src/app/components/trainers/trainer-info.ts
import {Star} from './star';
import {ReviewInfo} from '../profile/review-info';
import {SportResponse} from '../../services/sports/sport-response';
import {TrainerStatus} from '../../services/trainers/trainer-status';

export interface TrainerInfo {
  id: number;
  firstName: string;
  secondName: string;
  city: string;
  country: string;
  price: number;
  stars: Array<Star>;
  about: string;
  title: string;
  reviews: Array<ReviewInfo>;
  reviewTitle: string;
  trainingsCount: number;
  trainingsTitle: string;
  sports: Array<SportResponse>;
  status: TrainerStatus;
  photoUrl: string;
  backgroundUrl: string;
}

File: Sportlance.BLL/Entities/TrainerInfo.cs
using System.Collections.Generic;
using Sportlance.DAL.Entities;

namespace Sportlance.BLL.Entities
{
    public class TrainerListItem
    {
        public long Id { get; set; }

        public string FirstName { get; set; }

        public string SecondName { get; set; }

        public string Country { get; set; }

        public string City { get; set; }

        public double Price { get; set; }

        public double? Score { get; set; }

        public string Title { get; set; }

        public int FeedbacksCount { get; set; }

        public int TrainingsCount { get; set; }

        //TODO поменять на AzureBlobStorage
        public string PhotoUrl { get; set; }

        public IReadOnlyCollection<Sport> Sports { get; set; }
    }
}